extensions/roc-package-web-component-dev/src/commands/nunjucks.js

```diff
@@ -20,15 +20,17 @@ export default function nunjucksRendering() {
 
     // eslint-disable-next-line
     return function* demoRenderer() {
+        const templateVariables = {
+            name,
+            bundlePath: stats.script,
+            ...template.variables,
```

Comment:
Might it be better here to place all user defined variables under a single property? Like `userVariables` for example.